wordle-web/src/views/game.vue

```diff
@@ -0,0 +1,56 @@
+
+
+<template>
+  <div class="game">
+    <VCard variant="tonal">
```

Comment:
```suggestion
    <v-card variant="tonal">
```
Be consistent in how you reference the Vuetify components. Usually, we use a dash for the Vuetify components and pascal for our components to easily distinguish between our stuff and not our stuff.